pandas/tests/io/formats/test_style.py

```diff
@@ -40,6 +40,19 @@ def h(x, foo="bar"):
             ),
         ]
 
+        def bar_grad(a=None, b=None, c=None, d=None):
```

Comment:
can you define this as a module level function, its a bit odd to define it here. Why does it actually need to be part of self anyhow?

Reply:
what about putting it close to the `bar` tests as a `staticmethod` and no longer included in the `setup`? If you want outside the `TestStyler` class then I can put it at top of module. Ping if you need changing.

Reply:
yeah top of module is good. don't like staticmethods